django_executor/views.py

```diff
@@ -2,13 +2,17 @@
 from django.utils import timezone
 from django.http import JsonResponse
 from django.core.exceptions import PermissionDenied
-from django.core import urlresolvers
-from base import ManagementUtility, ManagementExecutor
-from models import Log
+try:
+    from django.core.urlresolvers import reverse
+except ImportError:
+    from django.urls import reverse
+
+from .base import ManagementUtility, ManagementExecutor
+from .models import Log
 
 
 def has_permission(request):
-    return request.user.is_authenticated() and request.user.is_superuser
+    return request.user.is_authenticated and request.user.is_superuser
```

Comment:
So this change would break Django 1.9
see https://github.com/django/django/commit/c1aec0feda7
I can add a try/except block if Django 1.9 support is still wanted.
Or I can bump the minimum Django version up.

However it wouldnt surprise me that `is_superuser` is already Django 1.10+ only

Reply:
I believe a try/except approach would be best. Would appreciate a PR.

Reply:
Just noting that only `is_superuser` is required in most cases, since https://github.com/django/django/commit/388182b622 which was the most usual case of `is_superuser` not working.

> However it wouldnt surprise me that is_superuser is already Django 1.10+ only

I was wrong - it was never a method - always a bool property as far as I can see.

> I believe a try/except approach would be best. 

Done.

Reply:
ping @mohammedhammoud & @hassanmian